src/api/IPAddress.js

```diff
@@ -0,0 +1,2 @@
+const IP_ADDRESS =  'http://192.168.1.110:8484';
```

Comment:
Also, just make one folder services/ and put your files in that.Remove api folder